scripts/dts/python-devicetree/src/devicetree/edtlib.py

```diff
@@ -71,7 +71,7 @@
 from copy import deepcopy
 from dataclasses import dataclass
 from typing import Any, Callable, Dict, Iterable, List, NoReturn, \
-    Optional, Set, TYPE_CHECKING, Tuple, Union
+    Optional, Pattern, Set, TYPE_CHECKING, Tuple, Union
```

Comment:
Apparently, `typing.Pattern` is deprecated. Can you use `re.Pattern` instead?

Reply:
good catch, fixed